content/projects/gsoc/2021/projects/conventional-commits-plugin.adoc

```diff
@@ -45,26 +44,29 @@ For a existing project released at `1.0.0`, the following commit messages would
 
 ==== Project Details
 
-This project idea proposes to implement a Jenkins plugin which will allow the `nextVersion` to be determined from the commit log.
+The conventional commits plugin is a Jenkins plugin to automatically determine the next semantic version for a git repository using the Conventional Commits (C.C) system.
 
-The project requires the student to start with plugin development from scratch and then work with understanding how to integrate this plugin into a Jenkins pipeline.
+Taking in various sources of input, like the commit history of the repository, the latest tagged version, and the version information read from various configuration files of the project, the plugin outputs the next semantic version.
```

Comment:
Where does it output the next semantic version? As a string from a function call? Can you provide a short code example here to get me interested?

Reply:
Yes sure, I will add the snippet